package.json

```diff
@@ -5,16 +5,18 @@
   "main": "index.js",
   "scripts": {
     "start": "NODE_ENV='development' npm run watch",
-    "build": "npm-run-all build:images-chrome build:js      build:pug      build:sass build:fonts build:Fafonts",
-    "build:beta": "npm-run-all build:images-chrome build:js:beta build:pug:beta build:sass build:fonts build:Fafonts",
+    "build": "npm-run-all build:images-chrome build:js            build:sass clean:chrome hash:chrome    build:pug  build:fonts build:Fafonts",
+    "build:beta": "npm-run-all build:images-chrome build:js:beta build:sass clean:chrome hash:chrome build:pug:beta build:fonts build:Fafonts",
     "build:js": "NODE_ENV=production parcel build src/js/chrome.js -d build/js --public-url /apps/chrome/js",
     "build:js:beta": "NODE_ENV=production parcel build src/js/chrome.js -d build/js --public-url /beta/apps/chrome/js",
     "build:images-chrome": "mkdir -p build/assets/images && ncp static/images build/assets/images",
-    "build:pug": "pug src/pug -o build/snippets -O \"{release: 'insights'}\"",
-    "build:pug:beta": "pug src/pug -o build/snippets -O \"{release: 'insightsbeta'}\"",
+    "build:pug": "chrome=$(ls build/ | grep chrome.*.css) && pug src/pug -o build/snippets -O \"{release: 'insights', chrome:'$chrome'}\"",
```

Comment:
@karelhala one question: do you mean move everything to the node file (including the compile pug commands)?

Reply:
Everything on these 2 lines, so it will be a bit nicer, not actually needed, but I am thinking about moving all these tasks to their node files, ao we might want to kick this off in this PR

Reply:
@karelhala Agree! I will start the node script, then. Thanks!